test/test.js

```diff
@@ -55,11 +46,11 @@ test.after.always(() => {
         throw err;
       }
     }
-    fs.writeFileSync('./test/out/failed-get-country.json', JSON.stringify(out, null, 2), err => {
+    fs.writeFileSync('./test/out/failed-country-code.json', JSON.stringify(out, null, 2), err => {
```

Comment:
writeFileSync -> outputJson
See [this](https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputJson.md)
Remove mkdirSync

Reply:
const failedPath = path.join('test', 'report', 'failed.geo.json');

Reply:
npm i --save-dev fs-extra